resolwe_bio/processes/differential_expression/dexseq.yml

```diff
@@ -0,0 +1,134 @@
+# ================================
+# Differential Expression - DEXSeq
+# ================================
+---
+
+- slug: differentialexpression-dexseq
+  name: DEXSeq
+  requirements:
+    expression-engine: jinja
+    executor:
+      docker:
+        image: resolwebio/rnaseq:2.1.1
+  data_name: 'Differential expression (case vs. control)'
+  version: 0.0.1
+  type: data:differentialexpression:dexseq
+  category: analyses
+  persistence: CACHED
+  description: >
+    Find differentially expressed exons.
+  input:
+    - name: case
+      label: Case
+      type: list:data:expression
```

Comment:
The type of expression data object is inconsistent with the run section of the process. If only expressions coming from _featureCounts_ are supported the input type should be set more specifically.